app/models/user.rb

```diff
@@ -16,9 +18,9 @@ class User < ActiveRecord::Base
   attr_accessible *(ACCESSIBLE_ATTRIBUTES + [:admin]), :as => :admin
 
   validates_presence_of :username
-  validates_uniqueness_of :username
+  validates :username, uniqueness: { case_sensitive: false }
```

Comment:
I'll add a migration with a warning, for existing users nothing changes. When two users have the same username only the [oldest](https://github.com/cantino/huginn/blob/master/app/models/user.rb#L38) is able to log in using the name.